schemas/team/summary.yml

```diff
@@ -0,0 +1,44 @@
+type: object
+
+properties:
+  id:
+    type: string
+    readOnly: true
+    description: |
+      The team's identifier (generated by the API)
+
+  url:
+    type: string
+    readOnly: true
+    description: |
+      API url for accessing the team description
+
+  title:
+    type: string
+    readOnly: true
+    description: |
+      The human-readable title of the team
+
+  members:
+    type: array
+    readOnly: true
+    items:
+      $ref: '#/definitions/user/summary'
+    description: |
+      Summaries of users that are part of this team.
+
+  organization:
+    $ref: '#/definitions/organization/summary'
+    readOnly: true
+    description: |
+      Summary of the organization that owns this team.
```

Comment:
We will probably need to display this information when displaying teams and their permissions for a project, but I think it does mean a change to seed-auth-api.